### PR TITLE
Reviewed Three Ruby Snippets

### PR DESCRIPTION
**Pull request by meg-gutshall**
I saw your open issue #28 and reviewed the first three Ruby snippets. The majority was spelling and grammar issues but I did add some code and documentation to two snippets just to bolster them. No code errors that I could find!

### CODE REVIEW

File: src/assets/snippets/ruby/array_count.md [hunk 1]
```diff
@@ -9,20 +9,27 @@ irb> numbers.count
...
 irb> numbers.count(&:even?)
  => 3
 ```
 
+The method above is basically shorthand for `numbers.count{|n| n.even?}`.
+
 ##### Additional links
 
-* [Ruby Doc - Array#count](https://ruby-doc.org/core-2.2.0/Array.html#method-i-count)
+* [Ruby Doc - `Array#count`](https://ruby-doc.org/core-2.2.0/Array.html#method-i-count)
```
Comment (florianjosefreheis) [inline src/assets/snippets/ruby/array_count.md]: Can we remove the `` to keep it consistent with the rest?
```suggestion
* [Ruby Doc - Array#count](https://ruby-doc.org/core-2.2.0/Array.html#method-i-count)
```

**Comment by florianjosefreheis** [comment]
Hi @meg-gutshall, thanks for the great work. Snippets look fine, just one small linting change and we are good from my side.

**Comment by meg-gutshall** [comment]
Yes, @florianjosefreheis that's fine with me! Glad to help!

**Comment by petrovicstefanrs** [comment]
@meg-gutshall @florianjosefreheis Is this PR ready to be merged? Or do you plan on adding some other changes? If not I'll merge it once you agree.

**Comment by florianjosefreheis** [comment]
@petrovicstefanrs ok from my side.

**Comment by meg-gutshall** [comment]
@florianjosefreheis @petrovicstefanrs Ready to merge. I will check other Ruby snippets as I have time and send them through. I'm hoping to do them in chunks of 3-4 at a time and then start creating snippets of my own once all are checked over.

**Comment by petrovicstefanrs** [comment]
@meg-gutshall @florianjosefreheis , Nice! Merging this and thank you both! 🚀

**Pull request merged**